examples/golang/components/sqldb-postgres-query/wadm.yaml

```diff
@@ -0,0 +1,45 @@
+---
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: go-sqldb-postgres-query
+  annotations:
+    version: v0.1.0
+    description: |
+      Demo WebAssembly component using the wasmCloud SQLDB Postgres provider via the wasmcloud:postgres WIT interface
+    wasmcloud.dev/authors: wasmCloud team
+    wasmcloud.dev/source-url: https://github.com/wasmCloud/wasmCloud/blob/main/examples/golang/components/sqldb-postgres-quer/wadm.yaml
+    wasmcloud.dev/readme-md-url: https://github.com/wasmCloud/wasmCloud/blob/main/examples/golang/components/sqldb-postgres-quer/README.md
+    wasmcloud.dev/homepage: https://github.com/wasmCloud/wasmCloud/tree/main/examples/golang/components/sqldb-postgres-quer
+    wasmcloud.dev/categories: |
+      database,sqldb,postgres,golang,example
```

Comment:
this must be a bug elsewhere from copy paste but it `sqldb-postgres-query` is missing the `y`

Reply:
whoops

Reply:
yeah this is definitely my bug that carried over here, I think I went back and fixed it for the other one